client/src/Components/Header/Header.jsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import { Box } from '@material-ui/core';
+import HeaderStyles from './HeaderStyles';
 
 const Header = () => {
-  return <div>Header</div>;
+  const imgSrc = 'https://files.slack.com/files-pri/TCB921BEE-F015MT9B5N3/logo.png';

```

Comment:
Umieścimy proszę to zdjęcie w repozytorium (src/assets/images)

Te zewnętrzne hostingi lubią robić problemy